.rubocop-rails.yml

```diff
@@ -102,6 +102,45 @@ Rails/Inquiry:
 Rails/WhereExists:
   Enabled: true
 
+Rails/ActiveRecordCallbacksOrder:
```

Comment:
Since we are already at version 2.7.0 of the rubocop-rails gem, and didn't change that all these cops are already enabled. By disabling them we would introduce a regression in rubocopital. 

Can we revert these disables?

Reply:
Reverted!